lib/shared/bucketing-assembly-script/assembly/index.ts

```diff
@@ -6,6 +6,7 @@ import { JSON } from 'assemblyscript-json/assembly'
 import { ConfigBody, DVCPopulatedUser, PlatformData } from './types'
 import { _clearPlatformData, _setPlatformData } from './managers/platformDataManager'
 import { _getConfigData, _setConfigData } from './managers/configDataManager'
+import { _getClientCustomData, _setClientCustomData } from './managers/globalDataManager'
```

Comment:
why are you calling it `clientCustomData` in some places but calling it `globalData` in others? Do you forsee the concept of shared data going beyond `client` scoped in the context of wasm?